v5/getting-started/macos.rst

```diff
@@ -12,7 +12,8 @@ The recommended method of installing PROS 3 for macOS involves using `Homebrew <
 1. If you do not already have Homebrew installed, install it by following the instructions on `their site <https://brew.sh>`_. This will take a while, and may prompt you to follow some additional instructions.
 2. Once you have Homebrew installed, run :code:`brew tap purduesigbots/pros` to register the PROS Homebrew repository with Homebrew.
 3. Run :code:`brew cask install gcc-arm-embedded pros-editor` to install the toolchain and the PROS Editor. This may also take a while.
-4. That's it! You can now start using PROS 3.
+4. (Optional) If you are planning to use the Vision Sensor, you will likely need to also install the VEX Vision Utility to set up signatures. Run :code:`brew install vcs-vision` to do this. Once installed, you'll be able to run the program by looking for "VCS vision" in searchlight.

```

Comment:
Instead of `brew cask install vcs-vision` it should be `brew cask install vcs-vision`